policyengine-client/src/countries/us/us.jsx

```diff
@@ -82,6 +102,32 @@ export class US extends Country {
                         "cdcc_min_rate",
                     ],
                 },
+                "Education": {
+                    "Phaseout": [
+                        "education_credit_phaseout_start_single",
+                        "education_credit_phaseout_start_joint",
+                        "education_credit_phaseout_length_single",
+                        "education_credit_phaseout_length_joint",
+                    ],
+                    "Lifetime Learning Credit": [
+                        "abolish_llc",
+                        "llc_max_expense",
+                    ],
+                    "American Opportunity Credit": [
+                        "abolish_aoc",
+                        "aoc_refundable_percentage",
+                    ],
+                },
+            },
+            "Payroll taxes": {
+                "Medicare": [
+                    "medicare_rate",
+                    "additional_medicare_rate",
+                ],
+                "Social Security": [
+                    "social_security_tax_threshold",
+                    "social_security_tax_rate",
+                ],
```

Comment:
Could you organize benefits by agency while in here? https://github.com/PolicyEngine/policyengine/issues/598